requirements-dev.txt

```diff
@@ -1,19 +1,23 @@
+azure-datalake-store >= 0.0.30
+azure-storage-blob >= 12.1.0
```

Comment:
I believe these should only be in the requirements-azure.txt file

Reply:
Sorry for this one.